extensions/granola/src/components/NoteComponents.tsx

```diff
@@ -0,0 +1,166 @@
+// External dependencies
```

Comment:
Unnecessary comments like this could probably be removed, just a suggestion though